doc/sphinx-guides/source/developers/testing.rst

```diff
@@ -95,6 +95,29 @@ You should be aware that some unit tests have been deemed "non-essential" and ha
 
 Generally speaking, unit tests have been flagged as non-essential because they are slow or because they require an Internet connection. You should not feel obligated to run these tests continuously but you can use the ``mvn`` command above to run them. To iterate on the unit test in Netbeans and execute it with "Run -> Test File", you must temporarily comment out the annotation flagging the test as non-essential.
 
+Integration Tests
+-----------------
+
+For integration testing we are using Arquillian, the example test you can look around at is called ``ArquillianExampleTest``.
```

Comment:
There is no 'look around at' idiom. There is 'look around for' but it means 'to search for' and I believe you you want just 'look at' or 'check out'.